src/data/finset/powerset.lean

```diff
@@ -241,5 +241,14 @@ begin
       { rwa [card_erase_of_mem hx, nat.lt_pred_iff] } } }
 end
 
+@[simp]
+lemma powerset_len_card_add (s : finset α) {i : ℕ} (hi : 0 < i) :
+  s.powerset_len (s.card + i) = ∅ :=
+finset.powerset_len_empty _ (lt_add_of_pos_right (finset.card s) hi)
+
+@[simp] theorem powerset_len_val (s : finset α) (i : ℕ) :
```

Comment:
This lemma name to me suggests the reverse direction to what you actually have.

Reply:
Ok I'll rename it to `map_val_val_powerset_len` I guess